drivers/usb/device/Kconfig

```diff
@@ -142,6 +142,22 @@ config USB_MCUX
 	help
 	  NXP MCUX USB Device Controller Driver for MXRT and LPC SoC's.
 
+config USB_MCUX_ENDPOINT_BUFFER_SIZE
+	int "Size of intermediate buffer in USB driver"
+	default 512
```

Comment:
I have not had time to look at it in detail, but this Kconfig option is problematic because it is not maximum by default, and a platform using this driver could require configuration overlay for some sample, subsequently I would block such overlay.

Reply:
I am curious what your rationale is for blocking a sample configuration overlay, but I will change this anyways

Reply:
actually I think I am just going to remove this Kconfig